tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/css/viewspecific/CreateMatter.css

```diff
@@ -1575,6 +1575,28 @@ left: -19px;
     -o-animation: spin 2s infinite linear;
     animation: spin 2s infinite linear;
 }
+.directiveFormFields{

```

Comment:
Settings files css changes to render the UI properly to fit new section of Matter extra fields.